### PR TITLE
Adding a picture to the mesh expansion and cross link

### PR DESCRIPTION
**Pull request by ldemailly**
Those pages are still very basic/wip sadly

### CODE REVIEW

File: _docs/guides/integrating-vms.md [hunk 1]
```diff
@@ -15,11 +15,16 @@ this infrastructure as a single mesh.
...
+  VMs cannot initiate any direct communication to Kubernetes Pods even when using Istio.
 
 ## Overview
 
-TBD
+
+<figure><img src="./img/mesh-expansion.svg" alt="BookInfo Application with Istio Mesh Expansion" title="BookInfo Application with Istio Mesh Expansion" />
+<figcaption>BookInfo Application with Istio Mesh Expansion</figcaption></figure>
+
+<!-- source of the drawing https://docs.google.com/drawings/d/1gQp1OTusiccd-JUOHktQ9RFZaqREoQbwl2Vb-P3XlRQ/edit -->
```
Comment (mandarjog) [inline _docs/guides/integrating-vms.md]: Can you ensure that this is in the shared folder ?

File: _docs/guides/integrating-vms.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Integrating Virtual Machines
 overview: This sample deploys the Bookinfo services across Kubernetes and a set of virtual machines, and illustrates how to use the Istio service mesh to control this infrastructure as a single mesh.
 
-draft: true
+#draft: true
```
Comment (mandarjog) [inline _docs/guides/integrating-vms.md]: why comment, why not remove ?

File: _docs/guides/integrating-vms.md (hunk 1 above)
Comment (ldemailly) [reply _docs/guides/integrating-vms.md]: done: "Mesh Expansion / Bookinfo" has been moved from My Drive to Presentations.

File: _docs/guides/integrating-vms.md (hunk 2 above)
Comment (ldemailly) [reply _docs/guides/integrating-vms.md]: done

**Pull request merged**